### DISCUSSION TITLE
Popover not work with Tab

### DISCUSSION THREAD

**Discussion by shana0440** [tailwindlabs/headlessui#1318] [Help]
Hi, I'm using @headlessui/react v1.5.0, and I'm try to using popover inside of tab, but when I click the popover panel, it will close automatically.

here is a reproduction URL:
https://codesandbox.io/s/tab-with-popover-yzbr8j-yzbr8j?file=/src/App.tsx

I'm wonder it is a bug or the popover is designed to not work with tab?

and thanks for the great works, I really enjoy this project.

**Comment by mohammadzolfali** [comment]
در تاریخ یکشنبه ۱۰ آوریل ۲۰۲۲،‏ ۲۲:۵۵ kw ***@***.***> نوشت:
